content/docs/command-reference/pull.md

```diff
@@ -56,6 +56,9 @@ After a data file is in cache, `dvc pull` can use OS-specific mechanisms like
 reflinks or hardlinks to put it in the workspace without copying. See
 `dvc checkout` for more details.
 
+Note that when you do `dvc pull` then the missing files whose corresponding DVC-files matches with the DVC-files in remote storage will be downloaded. But if a DVC-file is listed under `.dvcignore` then its corresponding file won't be downloaded.<br>
+See [.dvcignore](docs/user-guide/.dvcignore) for more details.
```

Comment:
Misisng a space and please don't add `<br>`s, just regular break lines
```suggestion
DVC might remove ignored files (files listed in `.dvcignore`) upon `dvc run` or `dvc repro`. If they are not produced by a pipeline stage, they can be deleted permanently. See [.dvcignore](docs/user-guide/.dvcignore) for more details.
```
It also needs formatting (max 80 char lines).